modules/time.js

```diff
@@ -0,0 +1,12 @@
+import { DateTime } from './config.js';
```

Comment:
According to the project requirements, instead of copying and pasting the luxon files, you are supposed to download the library into your project using `npm` and import it as a module.  eg: `import { DateTime } from "luxon"`. 
Kindly update your project accordingly :+1: 


![Screenshot from 2023-08-21 20-44-40](https://github.com/maximoortelli/Awesome-BookES6/assets/65233860/0b9a580b-1b93-4645-8365-8d01ccc097cb)

Reply:
Thank you very much for suggesting I have already made all the requested changes.